frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -307,84 +339,117 @@ function roleSelected(evt: any) {
+                                    numbers by separating them with
+                                    commas</small
+                                >
+                                <small
+                                    class="invalid-feedback"
+                                    v-if="
+                                        forestClientInactive ||
+                                        forestClientDuplicate
                                     "
-                                    @input="resetForestClientNumberData()"
-                                    :class="{
-                                        'is-invalid': errors.forestClientNumber,
-                                    }"
-                                ></InputText>
-                            </Field>
-                            <ErrorMessage
-                                class="invalid-feedback"
-                                name="forestClientNumber"
-                            />
-                        </div>
-
-                        <div class="col-md-2">
-                            <Button
-                                class="button"
-                                outlined
-                                aria-label="Verify forest client number"
-                                :name="'verifyFC'"
-                                :label="'Verify'"
-                                :loading-label="'Verifying...'"
-                                @click="
-                                    verifyForestClientNumber(
-                                        formData.forestClientNumber as string
-                                    )
-                                "
-                                v-bind:disabled="
-                                    formData.forestClientNumber?.length < 8 ||
-                                    LoadingState.isLoading.value
-                                "
-                            >
-                            </Button>
+                                >
+                                    {{
+                                        `Client ID ${
+                                            formData.forestClientNumber
+                                        }  ${
+                                            forestClientInactive
+                                                ? ' is inactive and cannot be added.'
+                                                : forestClientDuplicate
+                                                ? ' has already been added.'
+                                                : ''
+                                        }`
+                                    }}
+                                </small>
+                            </div>
+                            <div class="pr-1 col-2">
+                                <Button
+                                    outlined
+                                    aria-label="Verify forest client number"
```

Comment:
maybe the aria-label and label should matched?

Reply:
Fixed. Thanks.